packages/controller-utils/src/constants.ts

```diff
@@ -84,3 +84,40 @@ export const OPENSEA_PROXY_URL =
   'https://proxy.metafi.codefi.network/opensea/v1/api/v1';
 export const OPENSEA_API_URL = 'https://api.opensea.io/api/v1';
 export const OPENSEA_TEST_API_URL = 'https://testnets-api.opensea.io/api/v1';
+
+/**
+ * Approval request types for various operations.
+ * These types are used by different controllers to create and manage
+ * approval requests consistently.
+ */
+export const APPROVAL_TYPES = {
+  ADD_ETHEREUM_CHAIN: 'wallet_addEthereumChain',
+  ETH_ACCOUNTS: 'eth_accounts',
```

Comment:
Should this be an enum instead?

```suggestion
export enum ApprovalType {
  AddEthereumChain = 'wallet_addEthereumChain',
  EthAccounts = 'eth_accounts',
  // ...and so on
```

Reply:
No strong opinion on changing to enums here as you suggested. My initial thought was that TS enums are not a native JS feature, and `APPROVAL_TYPES` is meant to be used in the extension and mobile, however, both of the codes are being migrated to ts there is a long journey to it. That Is why I've not opted for an enum at first.

Reply:
All of the packages in this repo gets compiled into JavaScript in their published forms so that codebases (or parts of codebases) that aren't written in TypeScript can consume their APIs. Although the `enum` keyword and syntax is specific to TypeScript, enums were specifically designed to accommodate these kinds of situations, as TypeScript will produce an object during the compilation step. On the TypeScript side they also tend to be easier to work with than mere objects. With a plain object that acts like an enum, you need to define a type alongside that object (for example, here we'd need to define `type ApprovalType = typeof APPROVAL_TYPES[keyof typeof APPROVAL_TYPES]`), but with an enum you get that type for free. So I think is a great use of a enum.

Reply:
thanks to both of you :pray: 
I just submitted the changes.